src/test/scala/chiselTests/util/SRAMSpec.scala

```diff
@@ -33,13 +35,25 @@ class SRAMSpec extends ChiselFlatSpec {
     val (chirrtlCircuit, annos) = getFirrtlAndAnnos(new Top)
     val chirrtl = chirrtlCircuit.serialize
     chirrtl should include("module Top :")
-    chirrtl should include("smem sram_mem : UInt<8> [32]")
     chirrtl should include(
       "wire sram : { readPorts : { flip address : UInt<5>, flip enable : UInt<1>, data : UInt<8>}[0], writePorts : { flip address : UInt<5>, flip enable : UInt<1>, flip data : UInt<8>}[0], readwritePorts : { flip address : UInt<5>, flip enable : UInt<1>, flip isWrite : UInt<1>, readData : UInt<8>, flip writeData : UInt<8>}[1]}"
     )
+    chirrtl should include("mem _sram_MEM")
```

Comment:
The default name can't be `_sram_MEM`. The reason this changed is that `val mem` in `SRAM.scala` is of type `SyncReadMem` which gets handled by the compiler plugin. You can do the naming yourself by wrapping the RHS of `val sram = ` with `autoNameRecursively` [[1]](https://github.com/chipsalliance/chisel/blob/9177535ff1be47ffd99034bf0154c0f1ec637419/core/src/main/scala/chisel3/internal/plugin/package.scala#L30) (make sure to leave a comment explaining why you're doing this).